src/utils/openmctLocationSpec.js

```diff
@@ -32,7 +32,6 @@ import {
 import {resetApplicationState} from 'utils/testing';
 
 describe('the openmct location utility functions', () => {
-    beforeAll(() => resetApplicationState());
     afterEach(() => resetApplicationState());
```

Comment:
Should this be
`        return resetApplicationState(openmct);`

Reply:
I think its ok, this is without `{ }` so es6 auto returns it for arrow functions.

Reply:
@shefalijoshi Good catch. Normally yes, but in this case there's no instance of openmct to clean up. This test spec runs without Open MCT.